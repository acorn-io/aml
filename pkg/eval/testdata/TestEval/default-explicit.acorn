schema {
    foo: default 1
}

{}