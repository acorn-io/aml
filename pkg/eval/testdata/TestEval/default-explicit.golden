{
  "Foo": {
    "type": "object",
    "properties": {
      "foo": {
        "type": "number"
      }
    }
  },
  "foo": 1
}